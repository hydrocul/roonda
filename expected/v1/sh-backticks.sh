#!/bin/sh

cd "$ROONDA_TMP_PATH"
echo "`ls`"
