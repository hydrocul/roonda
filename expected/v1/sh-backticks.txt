roonda_2f5bd76c764b626d0c52cef7bfc5e4c336d449d8.sh
